content/03.results.md

```diff
@@ -131,25 +131,24 @@ Indeed, we observed significant upregulation (ANOVA p < 0.01) of the KRAS signal
 
 #### Diffuse astrocytic and oligodendroglial tumors
-In **Figure {@fig:Fig2}C**, we show genomic alterations in 61 HGGs biopsied at diagnosis: diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (HGGs, N = 26). <!--SAMPLECOUNT-->
-The single oligodendroglioma sample (N = 1) in the OpenPBTA does not contain mutations in the listed genes and is therefore not shown in this oncoprint.
+In **Figure {@fig:Fig2}C**, we show genomic alterations in 61 HGGs biopsied at diagnosis: diffuse midline gliomas (DMGs, N = 34) and other high-grade gliomas (non-midline HGGs, N = 27). <!--SAMPLECOUNT-->
 Across diffuse astrocytic and oligodendroglial tumors, we found that _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) were both most mutated and co-occurring genes (**Figure {@fig:Fig2}A**), followed by frequent mutations in _ATRX_ (30%, 18/61).
 We found recurrent amplifications and fusions in _EGFR_, _MET_, _PDGFRA_, and _KIT_, highlighting that these tumors utilize multiple oncogenic mechanisms to activate tyrosine kinases, as has been previously reported [@doi:10.1002/ijc.32258; @doi:10.1016/j.ccell.2017.08.017; @doi:10.1186/s40478-020-00905-w].
 Gene set enrichment analysis showed upregulation (ANOVA Bonferroni-corrected p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig5}B**).
 The two tumors with ultra-high TMB (> 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].
 
 #### Other CNS tumors
-**Figure {@fig:Fig2}D** depicts an oncoprint for the remaining primary CNS tumors in the OpenPBTA (N = 195). <!--SAMPLECOUNT-->
-We observed that 25% (15/60) of ependymoma tumors were _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion-positive ependymomas and that 70% (21/30) of craniopharyngiomas were driven by mutations in _CTNNB1_.
+**Figure {@fig:Fig2}D** depicts an oncoprint for the remaining primary CNS tumors in the OpenPBTA (N = 194). <!--SAMPLECOUNT-->
+We observed that 25% (15/60) of ependymoma tumors were _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion-positive ependymomas and that 68% (21/31) of craniopharyngiomas were driven by mutations in _CTNNB1_.
 Multiple histologies contained somatic mutations or fusions in _NF2_: 41% (7/17) of meningiomas, 5% (3/60) of ependymomas,  and 27% (3/11) schwannomas.
 Rare fusions in _ERBB4_, _YAP1_, _KRAS_, and _MAML2_ were observed in 10% (6/60) of ependymoma tumors.
 DNETs harbored alterations in MAPK/PI3K pathway genes as previously reported [@doi:10.1093/jnen/nlz101], including _FGFR1_ (21%, 4/19), _PDGFRA_ (10%, 2/19), and _BRAF_ (5%, 1/19).
-Frequent mutations in rarer brain tumor histologies (N < 5 samples in the OpenPBTA) are depicted in **Figure {@fig:S3}A**.
+Frequent mutations in additional, rare brain tumor histologies are depicted in **Figure {@fig:S3}A**.
```

Comment:
```suggestion
Frequent mutations in additional rare brain tumor histologies are depicted in **Figure {@fig:S3}A**.
```

Reply:
I think this belongs, since it is two adjectives together - they should be separated by a comma

Reply:
In this case, the two adjectives are different ranks, which is to say the order of the adjectives matters, so it is not needed. Another way of thinking about is that "additional" is modifying "rare brain tumors"